cal.html

```diff
@@ -0,0 +1,93 @@
+<html>
```

Comment:
Everything in one file? It works but with this much styling and code, should really be in separate files. It's a concept known as separation of concerns. Keep the different kinds of code (HTML, CSS, etc) away from each other in nice neat bundles.